src/JSInterop/Microsoft.JSInterop/src/JSInvokableAttribute.cs

```diff
@@ -43,6 +53,145 @@ public JSInvokableAttribute(string identifier)
             }
 
             Identifier = identifier;
+            Serializer = ParseArguments;
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="serializationContext"></param>
+        public JSInvokableAttribute(Type serializationContext)
```

Comment:
You'll want to add `DynamicallyAccessedMembers(PublicParameterlessConstructor)` to this type parameter.